sunkit_image/enhance.py

```diff
@@ -0,0 +1,122 @@
+"""
+This module contains functions that can be used to enhance the entire solar image.
+"""
+
+import numpy as np
+import scipy.ndimage as ndimage
+
+__all__ = [
+    "mgn",
+]
+
+
+def mgn(
+    data,
+    sigma=[1.25, 2.5, 5, 10, 20, 40],
+    k=0.7,
+    gamma=3.2,
+    h=0.7,
+    weights=None,
+    truncate=3
+):
+
+    """
+    Multi-scale Gaussian normalization.
+
+    Parameters
+    ----------
+    data : `numpy.ndarray`
+        Image to be transformed.
+    sigma : `list`, optional
+        Range of Gaussian widths to transform over.
+        Defaults to ``[1.25, 2.5, 5, 10, 20, 40]``.
+    k : `float`, optional
+        Controls the severity of the arctan transformation.
+        Defaults to 0.7
+    gamma : `float`, optional
+        The value used to calculate the  global gamma-transformed image.
+        Ideally should be between 2.5 to 4.
```

Comment:
Where do these numbers come from?

Reply:
These are the values which were recommended in the paper

Reply:
It is the recommended value in the paper

Reply:
We can mention that.